lib/amqp/gen/connection_manager.ex

```diff
@@ -14,8 +14,8 @@ defmodule Amqpx.Gen.ConnectionManager do
 
   @type state() :: %__MODULE__{}
 
-  def start_link(opts) do
-    GenServer.start_link(__MODULE__, opts, name: Amqpx.Gen.ConnectionManager)
+  def start_link(%{connection_params: connection_params} = opts) do
+    GenServer.start_link(__MODULE__, opts, name: Keyword.get(connection_params, :name) || Amqpx.Gen.ConnectionManager)
```

Comment:
why the `||` instead of the default argument?
```elixir
Keyword.get(connection_params, :name, Amqpx.Gen.ConnectionManager)
```

Reply:
I always use this pattern to take into account a value that is present but it's false-y, but since amqpx is a library maybe it's better to just use `Keyword.get/3` :+1: